.github/workflows/enterprise-pipeline.yml

```
@@ -0,0 +1,366 @@
+        run: |
+          echo "=== RUNNING ${{ matrix.test-group }} TESTS ==="
+          pytest tests/ \
+            -m "${{ matrix.test-group }}" \
+            --cov=. \
+            --cov-report=xml:coverage-${{ matrix.test-group }}.xml \
+            --cov-report=term-missing \
+            -v \
+            -n auto || true
+
+      - name: Upload coverage
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)